ITensorGPU/test/runtests.jl

```diff
@@ -1,6 +1,22 @@
-using ITensorGPU, Test, CUDA
+if VERSION < v"1.8"
+  itensorgpu_path = joinpath(pwd(), "..")
+  cp(
+    joinpath(pwd(), "..", "LocalPreferences.toml"),
+    joinpath("$(LOAD_PATH[2])", "JuliaLocalPreferences.toml"),
```

Comment:
Could you explain what `LOAD_PATH` is being used for here? Maybe there is a better way to do this, this seems fragile.